README.md

```diff
@@ -3,13 +3,11 @@ Manage GitHub team and repository configuration for DataWorks
 
 ## How to create a new GitHub repository for the DataWorks Project
 
-1. Note that at the moment you *must* use a UC Mac for the following procedure,
-   with access to the Crown Concourse infrastructure. Once Concourse moves to
-   AWS this restriction will be lifted.
 1. There are now two standards, for creating a new repository:
    1. _*Non-Terraform repositories:*_ Copy `template-repository.tf.sample` to a new `.tf` file to create a standard empty repository [1]
    1. _*Terraform repositories:*_ Copy `terraform-template-repository.tf.sample` to a new `.tf` file to create a standard Terraform repository, including Terraform templates and CI pipeline. [1]
-1. Update the `github_repository` resource name (example) using underscores as separators if required. Its `name`  should use hyphen separators if required, and `description` attributes should be a single sentence. *Do not* make any other changes.
+1. Update the `github_repository` resource name (example) using underscores as separators if required. Its `name`  should use hyphen separators if required, and `description` attributes should be a single sentence. 
+1. Replace every instance of `example` with the name of your repo using underscores.
```

Comment:
not quite, underscores for terraform resources and hyphens for repo name

Reply:
Which is what it says...?

Reply:
"Update the `github_repository` resource name (example) using underscores as separators if required"

Where `github_repository` is a tf resource?

"Its `name`  should use hyphen separators if required"

Which is the repo name?

Sorry I'm confused as to which part is wrong. All the other instances of `example` are tf resources or resource references, I believe.

Reply:
because I read this line without the context of the line above